packages/js/src/shared-admin/components/premium-upsell-card.js

```diff
@@ -15,17 +16,8 @@ import { ReactComponent as G2Logo } from "./g2-logo-white.svg";
  * @returns {JSX.Element} The premium upsell card.
  */
 export const PremiumUpsellCard = ( { link, linkProps } ) => {
-	const info = useMemo( () => createInterpolateElement(
-		sprintf(
-			/* translators: %1$s and %2$s expand to opening and closing <strong> tags. */
-			__( "Be more efficient in creating titles and meta descriptions with the help of AI. %1$sGet 24/7 support%2$s and boost your website’s visibility.", "wordpress-seo" ),
-			"<strong>",
-			"</strong>"
-		),
-		{
-			strong: <strong />,
-		}
-	), [] );
+	const info = useMemo( () => __( "Use AI to generate titles and meta descriptions, automatically redirect deleted pages, get 24/7 support and much, much more!", "wordpress-seo" ), [] );
+	const isBlackFriday = get( window, "wpseoScriptData.isBlackFriday", "" );
```

Comment:
Introducing a window dependency is a side-effect we should try to avoid by using the store. See other comment about the `preferences` reducer (I admit, the name is not that great).